auth-fail.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml">
+<head>
+    <meta charset="utf-8" />
+    <title>Authorization Failed</title>
+    <style>
+        .body {font-family:Verdana, Geneva, Tahoma, sans-serif }
+        .message {font-size: xx-large;}
+        .icon {height:100px; width:100px}
+    </style>
+</head>
+<body class="body">
+    <table>
+        <tr>
+            <td><img src="auth-fail.png" class="icon"/></td><td class="message">Authorization has Failed!</td></tr>
```

Comment:
This is super minor. Can we put the </tr> on the next line to line up with the <tr> at the top? Just trying to encourage readability and good formatting.

To update a Pull Request, just hop back onto the same branch, locally, that you submitted the commit on, and make the change. Then push those to your remote. It should update automatically here: https://stackoverflow.com/questions/9790448/how-to-update-a-pull-request-from-forked-repo

Otherwise, this looks good. I figured it'd give you the experience of going back and forth on a review.

Reply:
No problem in making changes. I can use the experience.

Looks like something was filtered out of your feedback, the tags you want me to move. I see "Can we put the on the next line to line up with the at the top". Looks like it should read "Can we put the A on the next line to line up with the B at the top"?

Reply:
haha...whoops, I guess I forgot to escape the characters. The '/tr' tag needs to line up with the opening 'tr' tag.